components/RecentStories.vue

```diff
@@ -34,7 +34,8 @@
 <script>
 import { mapState } from 'vuex'
 import disqus from '~/mixins/disqus'
-import helpers from '~/mixins/helpers'
+
+const { fuzzyDateTime } = require('~/mixins/helpers')
```

Comment:
you should be able to just use `import { fuzzyDateTime } from '~/mixins/helpers'` here